playbooks/common/openshift-cluster/upgrades/v3_10/upgrade_control_plane.yml

```diff
@@ -44,6 +44,9 @@
     # oo_lb_to_config might not be present; Can't use !oo_nodes because masters are nodes.
     l_sanity_check_hosts: "{{ groups['oo_etcd_to_config'] | union(groups['oo_masters_to_config']) | union(groups['oo_lb_to_config'] | default([]) ) }}"
 
+# Need to ensure ca.crt is valid
+- import_playbook: ../../../../init/validate_ca.yml
```

Comment:
Maybe it should be included in `sanity_checks.yml`? 

This way additional import in  playbooks/init/main.yml would not be required and CA validation can be skipped with `skip_sanity_checks`

Reply:
Done.